test/FeeManager.spec.ts

```diff
@@ -184,25 +195,31 @@ describe('Fee Manager', () => {
       RIF = vars.RIF;
       PartnerConfiguration = vars.PartnerConfiguration;
       PartnerManager = vars.PartnerManager;
+      partnerOwnerAccount = vars.partnerOwnerAccount;
 
       const depositAmount = oneRBTC.mul(5);
       const feePercentage = oneRBTC.mul(5);
 
       RIF.transfer.returns(true);
+      RIF.transferFrom.returns(true);
       await PartnerConfiguration.mock.getFeePercentage.returns(feePercentage);
       await PartnerManager.mock.getPartnerConfiguration.returns(
         PartnerConfiguration.address
       );
+      await PartnerManager.mock.getPartnerOwnerAccount.returns(
+        partnerOwnerAccount.address
+      );
 
       await expect(
         feeManager.connect(registrar).deposit(partner.address, depositAmount)
-      ).to.not.be.reverted;
+      ).eventually.fulfilled;
     });
 
     it('should withdraw successfully', async () => {
       try {
-        await expect(feeManager.connect(partner).withdraw()).to.not.be.reverted;
-        expect(await feeManager.balances(partner.address)).to.be.equals(
+        await expect(feeManager.connect(partnerOwnerAccount).withdraw()).to
+          .eventually.fulfilled;
+        expect(await feeManager.getBalance(partner.address)).to.be.equals(
```

Comment:
Should be checking the balance for `partnerOwnerAccount`

Reply:
yep